pallets/micropayment/src/lib.rs

```diff
@@ -503,7 +504,11 @@ pub mod pallet {
 
             let msg = Self::construct_byte_array_and_hash(server, nonce, session_id, amount);
 
-            let verified = sr25519_verify(&sig, &msg, &pub_key);
+            let verified = sr25519_verify(
```

Comment:
the `sr25519_verify` related discusses see below.

Reply:
" [0u8; 64];" default signature value  in pallets/micropayment/src/lib.rs "pub fn verify_signature",  it is pass the production check.